src/server/services/DummyService/dummy.cpp

```diff
@@ -32,7 +32,7 @@ class SampleService: public QgsService
     }
 
     void executeRequest( const QgsServerRequest& request, QgsServerResponse& response,
-                         QgsProject* project )
+                         const QgsProject* project )
```

Comment:
What is the meaning of a null ptr there ? If none, why not a const & rather than a const (naked) ptr ?